src/ci/docker/host-x86_64/disabled/riscv64gc-linux/0001-Remove-stime-function-calls.patch

```diff
@@ -89,8 +89,8 @@ index 70f829e7f..878375d78 100644
  				bb_perror_msg_and_die("can't set time of day");
 +		}
  	}
- 
+
  	if (flags != 1) /* not lone -s */
--- 
+--
 2.25.1
 
```

Comment:
Patch files shouldn't be changed IMO. These spaces are inserted by git, so removing them would mean that applying and recreating the patch would result in a different patch file.

Reply:
I delete changes from .patch files. Thank you!